src/main/java/com/cloudbees/hudson/plugins/folder/Folder.java

```diff
@@ -163,235 +101,58 @@
     @CopyOnWrite
     protected transient volatile List<Action> transientActions = new Vector<Action>();
 
-    /**
-     * {@link View}s.
-     */
-    private /*almost final*/ CopyOnWriteArrayList<View> views;
-
-    /**
-     * Currently active Views tab bar.
-     */
-    private volatile ViewsTabBar viewsTabBar;
-
-    /**
-     * Name of the primary view.
-     */
-    private volatile String primaryView;
-
-    private transient /*almost final*/ ViewGroupMixIn viewGroupMixIn;
-
-    private DescribableList<FolderHealthMetric,FolderHealthMetricDescriptor> healthMetrics;
-
     public Folder(ItemGroup parent, String name) {
         super(parent, name);
         init();
```

Comment:
It could be moved to `AbstractFolder` since it must be called by all subclasses.